liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/GenerateFeaturesMojo.java

```diff
@@ -80,10 +80,9 @@ public class GenerateFeaturesMojo extends PluginConfigSupport {
 
     @Override
     protected void init() throws MojoExecutionException {
-        // @see io.openliberty.tools.maven.BasicSupport#init() skip server config
-        // setup as generate features does not require the server to be set up install
-        // dir, wlp dir, outputdir, etc.
-        this.skipServerConfigSetup = true;
+        // @see io.openliberty.tools.maven.BasicSupport#init()
+        // do not skip server config setup as generate features requires
+        // the server to be set up: install dir, wlp dir, outputdir, etc.
```

Comment:
This is a new requirement since we are generating to the server dir. Is the plan to re-enable skipping server config setup for the case where users apply the "generateToSrc" param?